addons/link_tracker/tests/test_merge_utm_campaign.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tools import mute_logger
+from odoo.tests.common import TransactionCase
+
+
+class TestMergeUtmCampaign(TransactionCase):
+    def setUp(self):
+        result = super(TestMergeUtmCampaign, self).setUp()
+        self.UtmCampaign = self.env['utm.campaign']
+        self.LinkTracker = self.env['link.tracker']
+        self.LinkTrackerClick = self.env['link.tracker.click']
```

Comment:
No need to assign those to `self` as you're not using them in your test methods.